### ISSUE TITLE
Autonomous Emotion Learning in Speech: A View of Zero-Shot Speech Emotion Recognition

### ISSUE THREAD

**Issue by Theodb** [Theodb/PhD-research-papers#5]
## TL;DR
The proposed methodology employs emotional dimensions as the attributes in the zero-shot learning paradigm, resulting in two phases of learning, namely attribute learning and label learning. Attribute learning connects the paralinguistic features and attributes utilising speech with known emotional labels, while label learning aims at defining unseen emotions through the attributes.
### Article Link
https://www.isca-speech.org/archive/Interspeech_2019/abstracts/2406.html
### Author
Xinzhou Xu and Jun Deng and Nicholas Cummins and Zixing Zhang and Li Zhao and Björn W. Schuller
### Conference
Interspeech 2019
## Key Takeaways
* 
* 

## Useful Code Snippets
```python
#Add your codes here
import numpy as np
import pandas as pd

```

## Useful Tools
* 
*
